webbpsf/optics.py

```diff
@@ -1184,12 +1191,18 @@ def __init__(self, instrument, include_oversize=False, **kwargs):
         self.instr_name = instrument.name
 
         # work out which name to index into the CV results with, if for NIRCam
+        is_nrc_coron = False
```

Comment:
Similarly, curious if you can explain the reasoning for moving this up here - I would think that the `is_nrc_coron` variable should be defined under the ` if instrument.name == 'NIRCam':` block?

Reply:
If `is_nrc_coron` is defined only within the `if instrument.name == 'NIRCam':` block, then the variable is undefined for other instruments. But there are instances where it checks this variable no matter what instrument is in use.